src/components/Footer/styles/_35-footer-social.scss

```diff
@@ -23,12 +23,22 @@
 .footer-social-item {
   @extend %social-icon;
 
-  &:hover {
+  @include hocus {
+    border-color: var(--colour-white);
     color: var(--colour-white);
   }
 }
 
 .footer-social-item__icon {
   height: 100%;
+  padding-bottom: calc(1.75 * var(--space-xs));
```

Comment:
simplify to `padding: calc(1.75 * var(--space-xs));`

Reply:
will do - discussion for another (less time-pressured) occasion but would be interested to discuss longhand vs. shorthand, when and where to use